doc/classes/Window.xml

```diff
@@ -333,6 +333,7 @@
 			<return type="bool" />
 			<description>
 				Returns [code]true[/code] if the window is currently embedded in another window.
+				[b]Note:[/b] This will return [code]false[/code] when "Embed Game on Next Play" is enabled, because it still is a separate native window. To check for that use [method Engine.is_embedded_in_editor].
```

Comment:
```suggestion
				[b]Note:[/b] This will return [code]false[/code] for the root window when "Embed Game on Next Play" is enabled, because it still is a separate native window. To check for that use [method Engine.is_embedded_in_editor].
```
Or similar, will try to improve the general wording for this when I have the time

Reply:
I think this change may add more confusion. This note mixes in the setting `Embed Game on Next Play` that isn't directly related. If I change the setting, the value returned doesn't change until I restart the project and then the setting is applied.

Changing `Embed Game on Next Play` to `Embed Game` would be more clear I think.

Reply:
I'm not sure what you mean? This is specifically about this and not embedded windows in general